app/views/contexts/section_summary_context.py

```diff
@@ -57,23 +57,30 @@ def __call__(
 
         page_title = self.get_page_title(title_for_location)
 
-        return {
+        summary_context = {
             "summary": {
                 "title": title,
                 "page_title": page_title,
                 "summary_type": "SectionSummary",
                 "answers_are_editable": True,
-                "sections": [
-                    {
-                        "title": title,
-                        "groups": summary.get("groups"),
-                    }
-                ],
                 "collapsible": summary.get("collapsible"),
                 "custom_summary": summary.get("custom_summary"),
             },
         }
 
+        if summary.get("custom_summary"):
+            summary_context["summary"]["custom_summary"] = summary["custom_summary"]
+
+        if summary.get("groups") and not summary_context.get("custom_summary"):
```

Comment:
This can just be `elif groups` the not summary check if implicit at that point.

Reply:
Updated 👍